operator/validator/controller.go

```diff
@@ -159,6 +156,7 @@ type controller struct {
 	nonCommitteeMutex      sync.Mutex
 
 	recentlyStartedValidators atomic.Int64
+	metadataLastUpdated       *hashmap.Map[string, time.Time]
```

Comment:
I don't see why this needs to be `hashmap` (threadsafe). this is used only within the same loop in only one go routine.